client/gutenberg/extensions/gif/editor.scss

```diff
@@ -1,4 +1,6 @@
 .wp-block-jetpack-gif {
+	overflow: visible;
```

Comment:
Please review this fix for the focus/hover outline cropping @jeffersonrabb. I'm not sure why it needs to be hidden in the style sass.

Reply:
Looks like the original `overflow:hidden;` declaration was unnecessary - it was a fix for an issue that has been resolved in a better way. I removed both which resolves the hover outline issue. https://github.com/Automattic/wp-calypso/pull/30669/commits/01a8e84b08156036b2ccaa3a844737188adef868